cv32e40s/tests/programs/custom/xsecure_test/xsecure_test.c

```diff
@@ -0,0 +1,185 @@
+#ifndef __XSECURE_TEST_H__
+#define __XSECURE_TEST_H__
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+//TODO: functions are fetched from the privilege test code. If these functions end up in a common function file, we should adopt this file to use the common functions.
+
+#define CPUADDR_CPUCTRL     0xBF0
```

Comment:
Should it be `CSRADDR_*`? Or maybe there is some meaning that I'm not getting.

Reply:
A good question! Didnt notice it myself. I have copied the code from xsecure_csr.c, so if it is supposed to be so in the xsecure_csr file, it is probably also supposed to be so in this file.